app/views/recipe_foods/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<h4>Add Recipe Food</h4>
+<%= form_with model: [@recipe, @recipe.recipe_foods.create] do |form|%>
+<div class="mb-3">
+  <%= form.label :food %>
+  <div>
+    <%= form.select :food_id, Food.all.collect {|food| [food.name, food.id]},{
+    include_blank: true} , class:"form-select" %>
+  </div>
+</div>
+<div class="mb-3">
+  <%= form.label :quantity %>
+  <div>
+    <%= form.number_field :quantity , class:"form-control" , required: true,
+    placeholder: '1', min: 1%>
+  </div>
+</div>
+
+<div class="mb-3"><%= form.submit "Add", class:"btn btn-dark" %></div>
+<%end%>
+<div class="mb-3">
```

Comment:
- [ ] Currently `add button` is not working
![Screenshot from 2022-03-25 16-23-17](https://user-images.githubusercontent.com/65860678/160140847-1d08e77e-d127-4107-aec2-6a957e9f43ee.png)

Reply:
@uwadonat this button seems to work fine for both of us, I've also tested on multiple devices and browsers and they all work fine, are you sure it doesn't work on your end?

Reply:
You are adding negative values, which is not allowed on purpose as you can not prepare food in a negative timeline. Using a positive value works.

Reply:
Ok. If this is the case. I think it should be better to add some validations. No need to have negative numbers as option.

Thanks.

Reply:
### OPTIONAL
- [ ] Nice to add validation for a negative value. But when ı try to add negative value ı am not get any message. It will be better if show the message to the user.